src/connectedmachine/setup.py

```diff
@@ -48,7 +48,7 @@
     description='Microsoft Azure Command-Line Tools ConnectedMachine Extension',
     author='Microsoft Corporation',
     author_email='azpycli@microsoft.com',
-    url='https://github.com/Azure/azure-cli-extensions/tree/main/src/connectedmachine',
+    url='https://github.com/Azure/azure-cli-extensions/tree/master/src/connectedmachine',
```

Comment:
In order to avoid racial discrimination and eliminate sensitive words, we have an announcement [link](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-sdk%2Fissues%2F2745&data=04%7C01%7CZhou.Xing%40microsoft.com%7C2468b047df534406946908da0767c04e%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637830439490159422%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=6eXwr7%2FOMjaViZ%2FryjQ08IBALSKCX7Pgl8ILFdOCq3w%3D&reserved=0) to rename the branch name of GitHub. 
So please use the main branch instead of the master branch